lighthouse-core/audits/dobetterweb/uses-http2.js

```diff
@@ -43,7 +43,7 @@ class UsesHTTP2Audit extends Audit {
 
       // Filter requests that are on the same host as the page and not over h2.
       const resources = networkRecords.filter(record => {
-        const requestHost = new URL(record._url).host;
+        const requestHost = record._parsedURL && record._parsedURL.host;
```

Comment:
okay then.

add a comment to explain why we use URL.hostname but ParsedURL.host ?  :)